mods/ts/rules/bridges.yaml

```diff
@@ -41,6 +45,8 @@ CABHUT:
 	Armor:
 		Type: Concrete
 	HitShape:
+	EditorTilesetFilter:
+		Categories: Bridge
```

Comment:
This and all the LOBRDG already inherit it via `^LowBridgeRamp`, so you can remove it here and from the LOBRDGes.